res/Sectors/M1105/Avereguar.txt

```diff
@@ -0,0 +1,340 @@
+# Generated by https://travellermap.com
+# 2018-11-05T09:34:25+00:00
+
+# Avereguar
+# 7,3
+
+# Name: Avereguar (Hv)
+
+# Milieu: M1105
+
+# Credits: Avereguar sector was designed by Mark Humphreys.
+
+# Author:    Mark Humphreys
+
+# Subsector A: Aqaroo
+# Subsector B: Uafost
+# Subsector C: 
+# Subsector D: 
+# Subsector E: Lonylua
+# Subsector F: Avereguar
+# Subsector G: 
+# Subsector H: 
+# Subsector I: Itrasku
+# Subsector J: 
+# Subsector K: 
+# Subsector L: 
+# Subsector M: 
+# Subsector N: 
+# Subsector O: 
+# Subsector P: 
+
+# Alleg: Hf  : "Hive FDA Uplift/Observation Site"
+# Alleg: HvFd: "Hive Federation"
+# Alleg: CsHv: "Hive Client State"
```

Comment:
To be consistent with other sectors, this allegiance name should be "Client State, Hive Federation"

Reply:
Done